unstructured_inference/inference/layoutelement.py

```diff
@@ -68,11 +70,13 @@ def from_lp_textblock(cls, textblock: TextBlock):
         text = textblock.text
         type = textblock.type
         score = textblock.score
-        return cls(x1, y1, x2, y2, text, type, prob=score)
+        return cls(x1, y1, x2, y2, "detectron2_lp", text, type, score)
```

Comment:
lets either put `"detectron2_lp" into a kwarg or set as a constant somewhere so it is clear to caller this is what will be used.

Reply:
Actually, think this requires more than that. Until now I've been hardcoding the name on code just called with certain models, but probably we need all our models to contain a "model_name" variable for simplicity.